examples/spotify_example.py

```diff
@@ -96,25 +105,30 @@
     print("Failed to launch spotify controller due to credential error")
     sys.exit(1)
 
-# Query spotify for active devices
-devices_available = client.devices()
-
-# Match active spotify devices with the spotify controller's device id
-for device in devices_available["devices"]:
-    if device["id"] == sp.device:
-        spotify_device_id = device["id"]
-        break
+# The chromecast device does not show up as part of the public API get devices
+# call until it starts playing. The only way to do so is to transfer playback
+# by call this endpoint that's not part of their public API.
```

Comment:
Hmmm. I think this is not correct.
It will should show up. I dont need any special http calls

Reply:
Odd. How long did it take to show up? Was it a Chromecast V1/V2? or Google Home/Mini? 
Did you use the auth token fetched by feeding your cookies to `spotify_token`?

Reply:
I maintain spotcast so I think I represent most of the users with a variety of cast devies